.github/workflows/test-upstream.yaml

```diff
@@ -9,11 +11,11 @@ on:
       upstream_repo:
         description: "Upstream linkml repository to test against"
         required: true
-        default: "linkml/linkml"
+        default: "iQuxLE/linkml"
```

Comment:
???

Reply:
> This PR is currently working, but it's testing against my fork [iQuxLE/linkml](https://github.com/iQuxLE/linkml/tree/migrate-to-uv) instead of the main linkml repository. Before merging, we need to coordinate with the upstream linkml project. This is a temporary solution (to test workflows will be working) until both repositories complete their uv migrations.
> 
> As additional point, it might be needed to add docker to the main repos toml, as this would otherwise be failing as its missing in the dependencies. However it was working before (with poetry and no docker in toml) which is a little mysterious to me.

Reply:
this was because this PR is from my fork, there was an issue that it would not run the gh-actions correctly from main (sry I dont fully remember) but I am aware of this